addons/mail/static/src/components/message/message.xml

```diff
@@ -66,7 +66,7 @@
                         <t t-if="!messageView.isSquashed">
                             <div class="o_Message_header d-flex flex-wrap align-items-baseline ml-2">
                                 <t t-if="messageView.message.author">
-                                    <strong class="o_Message_authorName o_Message_authorRedirect o_redirect o_cursor_pointer text-truncate" role="link" tabindex="0" t-on-click="messageView.onClickAuthorName" title="Open profile">
+                                    <strong class="o_Message_authorName o_Message_authorRedirect o_redirect o_cursor_pointer text-truncate" role="button" tabindex="0" t-on-click="messageView.onClickAuthorName" t-att-title="messageView.hasAuthorOpenChat ? messageView.authorTitleText : ''">
```

Comment:
This change is actually not really ok, what if we click on the partner for which we can't open chat? We lose the ability to open profile instead?

I don't really like the spec here.

Also like this `role="button"` and `tabindex="0"` should only be set if `hasAuthorOpenChat` is true with current choice, as well as `o_cursor_pointer`.

Side note: unrelated to this PR, class `o_Message_authorRedirect` looks useless.

Reply:
@odoongr can you please give your opinion on this regarding opening the chat?